python-sdk/tests/benchmark/infrastructure/terraform/vars.tf

```diff
@@ -25,5 +25,5 @@ variable "tfstate_bucket" {
 variable "gke_node_pool_machine_type" {
   type        = string
   description = "GCP machine type used for the GKE Node Pool"
-  default     = "n2-standard-8"  # 8 vCPU and 32GB RAM, more options at https://cloud.google.com/compute/vm-instance-pricing
+  default     = "n2-standard-16"  # 16 vCPU and 64GB RAM, more options at https://cloud.google.com/compute/vm-instance-pricing
```

Comment:
Why!

Reply:
Hey Kaxil, I have created the ticket for memory leak, which is the reason why we require higher config - https://github.com/astronomer/astro-sdk/issues/1058

Reply:
Let's solve that first

Reply:
As per my discussion with @kaxil We agreed to run the benchmark on 1KB to 1GB grid, which runs fine on the existing machine size of `n2-standard-8`